### PR TITLE
merging everyone's work

### PR DESCRIPTION
**Pull request by kylemac**
the only thing that gets dropped is the currently (on `origin/master`) skipped `css` test, which is being `skipped` anyways. since there were a lot of merge conflicts, i'd love to have extra eyes on this.

tests pass locally, we'll see what travis says... 

cc @declandewet @jenius 

also going forward, lets work off `origin/develop` and PR into that

### CODE REVIEW

**Comment by kylemac** [comment]
actually, test don't pass locally. i needed to clear all of the `fixtures/**/public` dirs, and I get the same error that travis is getting on the `dump_dirs` test. i'll look into it a bit later

**Comment by declandewet** [comment]
That's a strange error :confused: Anything strange happening in `helpers.compileFixture`?

File: test/config_errors.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import 'babel-core/register'
```
Comment (declandewet) [inline test/config_errors.js]: This line is unnecessary - see https://github.com/carrot/roots-mini/pull/9/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R56

File: lib/plugins/plugin_utils.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import glob from 'glob'
+import path from 'path'
+import SingleEntryDependency from 'webpack/lib/dependencies/SingleEntryDependency'
+
+export default {
+  getFilesFromGlob: (compiler, opts) => {
```
Comment (declandewet) [inline lib/plugins/plugin_utils.js]: This syntax could be more terse:

```
getFilesFromGlob(compiler, opts) {
 // ...
}
```

**Comment by hhsnopek** [comment]
just a note for the future before we release we should wrap everything to 80 chars

**Comment by kylemac** [comment]
:tennis: 

have a review @jenius. this includes your config work + @declandewet's tooling/testing work.

**Comment by kylemac** [comment]
going to go ahead and merge this. @jenius if you want to review after the fact, feel free and we can address the changes in future PRs

**Pull request merged**